paper-input-container.html

```diff
@@ -336,8 +344,7 @@
       }
 
       // notify add-ons
-      var addons = Polymer.dom(this.$.addOnContent).getDistributedNodes();
-      for (var a, i = 0; a = addons[i]; i++) {
+      for (var a, i = 0; a = this._addons[i]; i++) {
```

Comment:
I say meh to these abbreviations. They think they are so cool. What do they even mean?! :unamused: